src/openforms/submissions/tests/test_submission_report.py

```diff
@@ -403,7 +403,13 @@ def test_report_is_generated_in_same_language_as_submission(self):
 
         for component_type, localised_input in fields:
             with self.subTest(f"FormIO label for {component_type}"):
-                self.assertIn(localised_input, html_report)
+                if component_type == "selectboxes":
+                    values = localised_input.split("; ")
+                    for value in values:
+                        self.assertIn(value, html_report)
+                else:
+                    self.assertIn(localised_input, html_report)
```

Comment:
can't you update the `fields` pairs instead? If we're no longer displaying lists with semi-colons (which is a change in behaviour), the premise of the test no longer applies so the test needs to change. This approach makes the test even more unreadable :(

I think changing the respective line to:

```py
("selectboxes", "<li>This</li>"),
("selectboxes", "<li>That</li>"),
("selectboxes", "<li>The Other</li>"),
```
would already be sufficient

Reply:
Ah, yeah that is a 100 times beter..

I've changed the `fields` to reflect the expected output for selectboxes :)